style.css

```diff
@@ -0,0 +1,525 @@
+  color: #fff;
+  text-transform: uppercase;
+}
+
+.video-content__description {
+  width: 248px;
+  font-size: 13px;
+  line-height: 16px;
+  margin-bottom: 10px;
+  color: #fff;
+}
+
+/* stories-content */
+
+.stories-content {
+  width: 100%;
+  padding-bottom: 90px;
+}
+
+.stories-content__h4 {
+  font-size: 22px;
+  font-weight: bold;
+  line-height: 21px;
+  text-align: center;
+  text-transform: uppercase;
+  margin-bottom: 40px;
+}
+
+.stories-items {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+}
+
+.stories-items__item {
+  width: 277px;
+}
+
+.stories-item__image {
+  display: block;
+  width: 100%;
+  margin-bottom: 30px;
+}
+
+.stories-item__title {
+  font-size: 18px;
+  line-height: 17px;
+  font-weight: bold;
+  margin-bottom: 8px;
+}
+
+.stories-item__description {
+  font-size: 13px;
+  line-height: 16px;
+  margin-bottom: 18px;
+}
+
+.stories-item__description-extended {
+  width: 100%;
+  display: grid;
+  grid-template-columns: 0.9fr 1fr 1fr;
+}
+
+.stories-item__date {
+  font-size: 12px;
+  line-height: 11px;
+  font-weight: bold;
+  color: #000;
+  opacity: 0.3;
+  text-align: right;
+}
+
+/* gallery */
+
+.gallery {
+  width: 100%;
+  padding-bottom: 94px;
+}
+
+.gallery__title {
+  font-size: 18px;
+  line-height: 17px;
+  color: #000;
+  opacity: 0.3;
+  text-align: right;
+  text-transform: uppercase;
+  font-weight: bold;
+  margin-bottom: 8px;
+}
+
+.gallery-content {
+  display: grid;
+  grid-template-columns: repeat(7, 1fr);
+}
+
+.gallery-content__item {
+  display: block;
+}
+
+.gallery__photo {
+  width: 100%;
+}
+
+.hidden-1024 {
+  display: none;
+}
+
+.footer {
+  background-color: #000;
+  background-size: cover;
+  font-family: 'GothamPro', serif;
+  margin: 0 auto;
+  box-sizing: border-box;
+}
+
+.footer__content {
+	max-width: 930px;
+  margin: 0 auto;
+  padding-top: 69px;
+  display: flex;
+  justify-content: space-around;
+}
+
+.footer__column {
+  display: block;
+}
+
+.footer__column-title {
+	margin-bottom: 10px;
+	color: #fff;
+}
+
+.footer-column__nav {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: left;
+}
+
+.footer-column__nav-link {
+	display: inline-block;
+  text-decoration: none;
+  color: #fff;
+  font-size: 12px;
+  line-height: 11px;
+  margin-bottom: 10px;
+}
+
+.footer__bottom {
+	width: 100%
+}
+
+.copyright {
+	color: #fff;
+	font-size: 12px;
+  line-height: 11px;
+  text-align: center;
+  padding-top: 60px;
+  padding-bottom: 33px;
+}
+
+.footer__button {
+	border: 1.5px solid #fff;
+  background: transparent;
+  width: 137px;
+  height: 30px;
+  margin-top: 11px;
+  margin-bottom: 37px;
+}
+
+.button__link {
+	font-family: 'GothamPro', serif;
+	font-size: 10px;
+  line-height: 10px;
+  color: #fff;
+  text-transform: uppercase;
+  text-decoration: none;
+  font-weight: bold;
+}
+
+.socials {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+}
+
+.socials__item {
+	display: block;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/PolinaYartseva_axion/pullRequest?prid=5569682)